diktat-rules/src/main/kotlin/com/saveourtool/diktat/ruleset/rules/chapter2/kdoc/KdocMethods.kt

```diff
@@ -197,11 +201,42 @@ class KdocMethods(configRules: List<RulesConfig>) : DiktatRule(
                 && !hasReturnKdoc && !isReferenceExpressionWithSameName
     }
 
+    private fun isThrowInTryCatchBlock(node: ASTNode?): Boolean {
+        node ?: return false
+        var parent = node.treeParent
+        while (parent != null && parent.elementType != TRY) {
+            parent = parent.treeParent
+        }
+        val nodeName = node.findAllDescendantsWithSpecificType(IDENTIFIER).firstOrNull() ?: return false
+
+        if (parent?.elementType == TRY) {
+            val catchNodes = parent.getAllChildrenWithType(CATCH)
+            val findNodeWithMatchingName = catchNodes.firstOrNull { catchNode ->
+                val matchingNameForCatchNode = catchNode.findAllDescendantsWithSpecificType(IDENTIFIER)
```

Comment:
Maybe it's worth changing names of variables to take into account new logic that takes into account child elements of exceptions, and not just matching of their names?

Reply:
Done